README.md

```diff
@@ -263,6 +263,9 @@ expect(screenshots).to.not.matchReference();
 ## Nightwatch Utility
 Nightwatch.js is an easy to use Node.js based End-to-End (E2E) testing solution for browser based apps and websites. It uses the powerful W3C WebDriver API to perform commands and assertions on DOM elements. Full documentation regarding nightwatch can be found at http://nightwatchjs.org/.
 
+### Setup
```

Comment:
Given nightwatch & wdio will need docker set up, it might be better to add the docker requirement under the Getting Started Section